pkg/apis/config/defaults.go

```diff
@@ -137,6 +137,8 @@ type Defaults struct {
 
 	UserContainerNameTemplate string
 
+	ContainerConcurrency *int64
```

Comment:
You need to add logic to the method above to parse this, and should add it to the `_example` block of the configmap.

Reply:
I also don't think this should be a pointer, just an `int64` that defaults to `0` (what we have today).